man/sd_bus_message_dump.xml

```diff
@@ -26,18 +27,25 @@
     <funcsynopsis>
       <funcsynopsisinfo>#include &lt;systemd/sd-bus.h&gt;</funcsynopsisinfo>
 
+      <funcsynopsisinfo><token>enum</token> {
+      <constant>SD_BUS_MESSAGE_DUMP_WITH_HEADER</constant>,
+      <constant>SD_BUS_MESSAGE_DUMP_SUBTREE_ONLY</constant>
```

Comment:
This is kinda misleading as it seems to suggest that these are not flags but enum values in sequence?